tests/ts_util.erl

```diff
@@ -138,19 +139,19 @@ ts_insert(Conn, Table, Columns, Data) ->
     ValClause = string:strip(lists:foldl(TermFn, [], tuple_to_list(Data)), right, $,),
     SQL = flat_format("INSERT INTO ~s (~s) VALUES (~s)",
                       [Table, ColClause, ValClause]),
-    lager:info("~ts", [SQL]),
+    % lager:info("~ts", [SQL]),
```

Comment:
Ooops. I didn't mean to commit that, do we need to log though? I think this is a side effect of building the SQL dynamically instead of plain text.  The plain text bulks up the test build will not be read as often as the test logs.